r4/src/main/java/org/opencds/cqf/r4/providers/PlanDefinitionApplyProvider.java

```diff
@@ -132,9 +122,40 @@ private void resolveDefinition(Session session, PlanDefinition.PlanDefinitionAct
             logger.debug("Resolving definition " + action.getDefinitionCanonicalType().getValue());
             String definition = action.getDefinitionCanonicalType().getValue();
             if (definition.startsWith(session.getPlanDefinition().fhirType())) {
-                logger.error("Currently cannot resolve nested PlanDefinitions");
-                throw new NotImplementedException(
-                        "Plan Definition refers to sub Plan Definition, this is not yet supported");
+                IdType id = new IdType(definition);
+                CarePlan plan = null;
+                try {
+                    plan = applyPlanDefinition(id,
+                            session.getPatientId(),
+                            session.getEncounterId(),
+                            session.getPractitionerId(),
+                            session.getOrganizationId(),
+                            session.getUserType(),
+                            session.getUserLanguage(),
+                            session.getUserTaskContext(),
+                            session.getSetting(),
+                            session.getSettingContext());
+                } catch (IOException e) {
+                    e.printStackTrace();
+                } catch (JAXBException e) {
```

Comment:
```suggestion
                } catch (IOException | JAXBException e) {
```

Since the catch branches are identical we can [collapse them](https://docs.oracle.com/javase/7/docs/technotes/guides/language/catch-multiple.html)